### PR TITLE
Correct 'one-var' linter rule

### PR DESCRIPTION
**Pull request by autra**
I'm continuing this linter effort. Here with the `one-var` rule.

@iTowns/reviewers please R?

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -54,7 +55,6 @@ module.exports = {
     'no-param-reassign': 'off',
     'no-else-return': 'off',
     // the following should be reactivated with 'one-var': 'never'
```
Comment (tbroyer) [inline .eslintrc.js]: That line should be removed

File: src/Core/Math/CVML.js [hunk 2]
```diff
@@ -2009,11 +1997,11 @@ CVML.rep = function rep(s, v, k) {
 };
 
 CVML.transpose = function transpose(A) {
-    var ret = [],
-        /* n = A.length,*/
-        i,
-        j,
-        Ai;
+    const ret = [];
+    /* let n = A.length,*/
```
Comment (tbroyer) [inline src/Core/Math/CVML.js]: The comma could be changed to a semicolon here :wink:

File: src/Core/Math/CVML.js [hunk 3]
```diff
@@ -2356,21 +2344,17 @@ CVML.svd = function svd(A) {
...
-    q = y.length;
-    r = y[0].length;
-    ret = Array(p);
+    let i;
+    let j;
+    let k;
+    let foo;
+    let bar;
+    let woo;
+    let i0; // ,k0,p0,r0;
```
Comment (tbroyer) [inline src/Core/Math/CVML.js]: Remove the `,k0,p0,r0` or make separate (commented out) declarations?

File: src/Core/Math/CVML.js [hunk 4]
```diff
@@ -2427,25 +2411,22 @@ CVML.dotMMbig = function dotMMbig(x, y) {
...
-        k,
-        p,
-        q, /* r,*/
-        ret, /* foo,bar,*/
-        woo,
-        i0;
-    p = x.length;
-    q = y[0].length;
-    ret = Array(q);
+    let /* i,*/ j;
```
Comment (tbroyer) [inline src/Core/Math/CVML.js]: Same here (remove `i` or make separate –commented out– declaration)

**Review by tbroyer — CHANGES_REQUESTED** [review]

File: src/Core/Math/CVML.js (hunk 2 above)
Comment (gchoqueux) [reply src/Core/Math/CVML.js]: for information CVML.js is removed in PR #195

Comment (autra) [reply src/Core/Math/CVML.js]: I don't expect #195 to be merged very fast, looking at its size...

**Comment by autra** [comment]
updated @iTowns/reviewers 

I've preferred to delete comments than correct them. They seem to be outdated anyway.

**Review by tbroyer — APPROVED** [review]

**Pull request merged**